apps/web/src/App.tsx

```diff
@@ -154,218 +151,216 @@ function App() {
       <HelmetProvider>
         <BrowserRouter basename={CONTEXT_PATH}>
           <QueryClientProvider client={queryClient}>
-            <AuthHandlerComponent>
-              <ThemeHandlerComponent>
-                <SpotLightProvider>
-                  <Routes>
-                    <Route path={ROUTES.AUTH_SIGNUP} element={<SignUpPage />} />
-                    <Route path={ROUTES.AUTH_LOGIN} element={<LoginPage />} />
-                    <Route path={ROUTES.AUTH_RESET_REQUEST} element={<PasswordResetPage />} />
-                    <Route path={ROUTES.AUTH_RESET_TOKEN} element={<PasswordResetPage />} />
-                    <Route path={ROUTES.AUTH_INVITATION_TOKEN} element={<InvitationPage />} />
-                    <Route path={ROUTES.AUTH_APPLICATION} element={<CreateOrganizationPage />} />
+            <AuthProvider>
+              <SpotLightProvider>
```

Comment:
I think that the spotlight shouldn't work on the non registered pages, maybe we can move it to be only part of the layout?

Reply:
this is only storing actions for the spotlight that is wrapping each route...